failed pack dependency installation

my setup is running into the same error each time, can't figure out why it is failing on the Ansible installation pack

```
==> chatops: id: 59ad556e0a84b422cb842b77
==> chatops: action.ref: packs.install
==> chatops: parameters: 
==> chatops:   packs:
==> chatops:   - ansible
==> chatops: status: failed
==> chatops: result_task: install pack dependencies
==> chatops: result: 
==> chatops:   exit_code: 1
==> chatops:   result: None
==> chatops:   stderr: "st2.actions.python.SetupVirtualEnvironment

------

==> chatops: traceback: None
==> chatops: failed_on: install pack dependencies
==> chatops: start_timestamp: 2017-09-04T13:30:22.129460Z
==> chatops: end_timestamp: 2017-09-04T13:30:55.177667Z
==> chatops: +--------------------------+------------------------+------+--------+-----------------+
==> chatops: | id                       | status                 | task | action | start_timestamp |
==> chatops: +--------------------------+------------------------+------+--------+-----------------+
==> chatops: | 59ad556e0a84b41f320e2bcc | succeeded (4s elapsed) | down | packs. | Mon, 04 Sep     |
==> chatops: |                          |                        | load | downlo | 2017 13:30:22   |
==> chatops: |                          |                        | pack | ad     | UTC             |
==> chatops: | 59ad55720a84b41f320e2bcf | succeeded (2s elapsed) | make | packs. | Mon, 04 Sep     |
==> chatops: |                          |                        | a pr | virtua | 2017 13:30:26   |
==> chatops: |                          |                        | erun | lenv_p | UTC             |
==> chatops: |                          |                        |      | rerun  |                 |
==> chatops: | 59ad55740a84b41f320e2bd2 | failed (26s elapsed)   | inst | packs. | Mon, 04 Sep     |
==> chatops: |                          |                        | all  | setup_ | 2017 13:30:28   |
==> chatops: |                          |                        | pack | virtua | UTC             |
==> chatops: |                          |                        | depe | lenv   |                 |
==> chatops: |                          |                        | nden |        |                 |
==> chatops: |                          |                        | cies |        |                 |
==> chatops: +--------------------------+------------------------+------+--------+-----------------+
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.
````

I think it have to do something with ```.h:17:27: fatal error: gssapi/gssapi.h: No such file or directory``` but don't know for sure

--- a/ansible.sh
+++ b/ansible.sh
@@ -4,6 +4,9 @@ set -e
 echo "#############################################################################################"
 echo "########################### Prepare StackStorm and Ansible ##################################"
 
+# Install Ansible core dependencies
+sudo apt-get install -y gcc libkrb5-dev
+
 # Install ansible integration pack
 st2 pack install ansible
 

